data/config/packages.json

```diff
@@ -1417,23 +1427,58 @@
 				]
 			}
 		},
-		"openrazer": {
+		"polychromatic": {
 			"packages": [
-				"openrazer-daemon"
+				"polychromatic", "openrazer-daemon"
 			],
 			"repos": {
 				"all": [
-					"ppa:openrazer/stable"
+					"ppa:polychromatic/stable", "ppa:openrazer/stable"
+				]
+			},
+			"post-install": "openrazer-setup"
+		},
+		"openrazer-setup": {
+			"script": "openrazer-setup"
+		},
+		"discord": {
+			"snap": [
+				"discord"
+			]
+		},
+		"obs-studio": {
+			"snap": [
+				"obs-studio"
+			]
+		},
+		"mangohud": {
+			"packages": [
+				"mangohud"
+			],
+			"repos": {
+				"all": [
+					"ppa:flexiondotorg/mangohud"
 				]
 			}
 		},
-		"polychromatic": {
+		"corectrl": {
+			"packages": [
+				"corectrl"
+			],
+			"repos": {
+				"all": [
+					"ppa:ernstp/mesarc"
+				]
+			}
+		},
+		"piper": {
 			"packages": [
-				"polychromatic"
+				"ratbagd", 
+				"piper"
 			],
 			"repos": {
 				"all": [
-					"ppa:polychromatic/stable"
+					"ppa:libratbag-piper/piper-libratbag-git"
```

Comment:
Again - similar issue - no jammy packages

Reply:
drop for now ?

Reply:
So for both packages Mauro helped me find them on the universe package with jammy support, so replaced in packages.json